components/private_channel/rust/ffi/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
+cpp.out
```

Comment:
I think this should be updated to `cc.out`, you have `components/private_channel/rust/ffi/tests/cc.out` included in the PR

Reply:
@antonok-edm see https://github.com/brave/brave-core/pull/5765/files#r508025787 and https://github.com/brave/brave-core/pull/5765/files#r508021757
we should not be building this stuff outside of gn in the first place and we should not be generating outputs in the source directory

Reply:
I removed the `Makefile` and all building-related configs from the PR.